### ISSUE TITLE
Skeleton Delete Acount Test Fails at very create project

### ISSUE THREAD

**Issue by katsuren** [laravel/framework#36931]
- Laravel Version: 8.36.2
- PHP Version: PHP 8.0.3 (cli) (built: Mar  5 2021 07:54:13) ( NTS )
- Database Driver & Version: mysql  Ver 8.0.23-0ubuntu0.20.04.1 for Linux on x86_64 ((Ubuntu))

### Description:

In the project that I created, run phpunit result fail.

```

  • Tests\Feature\DeleteAccountTest > user accounts can be deleted
   RuntimeException 

  Session store not set on request.

  at vendor/laravel/framework/src/Illuminate/Http/Request.php:483
    479▕      */
    480▕     public function session()
    481▕     {
    482▕         if (! $this->hasSession()) {
  ➜ 483▕             throw new RuntimeException('Session store not set on request.');
    484▕         }
    485▕ 
    486▕         return $this->session;
    487▕     }

      +32 vendor frames 
  33  tests/Feature/DeleteAccountTest.php:27
      Livewire\Testing\TestableLivewire::call()
```

### Steps To Reproduce:

```
laravel new project --jet
# choose livewire
# dont use team feature

cd project
npm install
npm run dev

php artisan test
```

**Comment by katsuren** [comment]
phpunit.xml sttings is below

using database on memory and connection sqlite

```
<?xml version="1.0" encoding="UTF-8"?>
<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="./vendor/phpunit/phpunit/phpunit.xsd"
         bootstrap="vendor/autoload.php"
         colors="true"
>
    <testsuites>
        <testsuite name="Unit">
            <directory suffix="Test.php">./tests/Unit</directory>
        </testsuite>
        <testsuite name="Feature">
            <directory suffix="Test.php">./tests/Feature</directory>
        </testsuite>
    </testsuites>
    <coverage processUncoveredFiles="true">
        <include>
            <directory suffix=".php">./app</directory>
        </include>
    </coverage>
    <php>
        <server name="APP_ENV" value="testing"/>
        <server name="BCRYPT_ROUNDS" value="4"/>
        <server name="CACHE_DRIVER" value="array"/>
        <server name="DB_CONNECTION" value="sqlite"/>
        <server name="DB_DATABASE" value=":memory:"/>
        <server name="MAIL_MAILER" value="array"/>
        <server name="QUEUE_CONNECTION" value="sync"/>
        <server name="SESSION_DRIVER" value="array"/>
        <server name="TELESCOPE_ENABLED" value="false"/>
    </php>
</phpunit>
```

**Comment by katsuren** [comment]
It can delete user by browser, so I think it is due to way to test

**Comment by driesvints** [comment]
This is fixed in the next release

**Comment by katsuren** [comment]
This is not fixed in version 8.37.0

**Comment by driesvints** [comment]
@katsuren Jetstream still needs to be tagged

**Comment by katsuren** [comment]
OK, describe jetstream version, just for record.
current jetstream version = 2.3.1

**Comment by driesvints** [comment]
@katsuren I just released v2.3.2: https://github.com/laravel/jetstream/releases/tag/v2.3.2

**Comment by katsuren** [comment]
@driesvints In Jetstream v2.3.2
This is fixed, thank you!
